src/pages/about.tsx

```diff
@@ -1,13 +1,41 @@
 import { Gallery } from '@/components/Blocks/Gallery/Gallery';
+import type { Icon } from '@/components/Blocks/IconSet/components/Icon';
 import { IconSet } from '@/components/Blocks/IconSet/IconSet';
 import styles from '@/styles/pages/About.module.scss';
 
 export default function About(): JSX.Element {
+    const teamIcons: Icon[] = [
+        {
+            image: {
+                src: '/images/man-profile-2.png',
+                alt: 'Meetup Logo',
+            },
+            title: <div className={'h3 ' + styles.name}>Fred Lawler</div>,
+            subtitle: <div className={'h6 ' + styles.position}>Co-Organizer</div>,
```

Comment:
I'm curious if this is more semantic: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure

Reply:
In general, if you need block level elements, perfer `<p>` or something else. Divs should be reserved for ad-hoc structure where other block elements don't make sense.

Reply:
These are both really helpful! I will be implementing them.